### PR TITLE
[Documentation] update readme

### PR DESCRIPTION
**Pull request by mathildemerle**
This PR update the readme:
 * Asclepios -> Epione. By the way, Visages changed its name also, isn't it?
 * Remove the compilation steps to centralize on the final documentation. At the end, it would be on medInria website i think, but for now i put the new dev guide on Florent's github. It could be changed quickly as soon as a new one appears.
 * markdown links.
 * the copyright.

edit: guide dev issue: https://github.com/medInria/medInria-public/issues/498

:m:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,32 @@
 # medInria
 
-is a medical image visualisation and processing tool developed at Inria by the teams Asclepios, Athena, Parietal and Visages.
+is a medical image visualisation and processing tool developed at Inria by the teams Epione, Athena, Parietal and Visages.
```
[the remark this answers is not in the document]
Comment (Florent2305) [reply README.md]: Did we keep both name the old and the new?

[the remark this answers is not in the document]
Comment (mathildemerle) [reply README.md]: Something like "Epione (ex Asclepios)" and "Empenn (ex Visages)" ?

**Comment by Florent2305** [comment]
I think it's a good idea to ask Maxime or Olivier about the validity of Athena and Parietal names

**Comment by msermesant** [comment]
we keep them. I updated the medInria website last week.

**Comment by mathildemerle** [comment]
The readme PR is updated with ex team names

**Pull request merged**